data/sql/updates/db_auth/2021_05_13_00.sql

```diff
@@ -38,7 +38,7 @@ CREATE TEMPORARY TABLE `_temp_base32_lookup1`
     `c` char(1) not null,
     `v` tinyint unsigned not null,
     primary key (`c`)
-);
+)DEFAULT CHARSET=utf8mb4;
```

Comment:
I will just leave this sneaky sql thing in here =)

Reply:
?

Reply:
Just to prevent coallation issue, it was supposed to be on another branch but github fucked me and Im too lazy to open a new PR for it :D